### ISSUE TITLE
External resources workflow failing due to missing vaccine platforms

### ISSUE THREAD

**Issue by agitter** [greenelab/covid19-review#1101]
Our external resources GitHub Actions workflow has been failing for about 2 months.
- [Last good run](https://github.com/greenelab/covid19-review/runs/4829438178?check_suite_focus=true)
- [First bad run](https://github.com/greenelab/covid19-review/runs/4834925613?check_suite_focus=true)

It happens in the "Update OWID data" step:
```
Generating Our World in Data COVID-1[9](https://github.com/greenelab/covid19-review/runs/4834925613?check_suite_focus=true#step:7:9) vaccine statistics
Missing platform information for Novavax
Error: Process completed with exit code 1.
```

Based on this comment, it looks like `vaccine_platforms.csv` needs to be manually updated.  Is that correct @rando2?
https://github.com/greenelab/covid19-review/blob/b493cdf544a0ea04d3feb53a908a21790fcb457b/owiddata/generate-owiddata-stats.py#L106-L109

In the latest runs, there are multiple missing platforms.

**Comment by rando2** [comment]
Thank you @agitter! I wasn't sure how to handle this because I couldn't find a machine readable list of the vaccine types. I don't know if there's a better way to throw an error that would be less disruptive to the project as a whole?

**Comment by agitter** [comment]
The external resources build worked again 02c8aeecd984a4c050c49f1b8b85a61ab0603c7b

We can close this or leave it open if you want to work on the automated HTML parsing.

**Comment by rando2** [comment]
Hopefully a permanent solution is proposed in #1104! I worry a little because the way the HTML is structured suggests that it could change (for example, if they decide to change the text size, the script would probably break because the classes used just say what the text size is rather than what it contains). But for now this is a better solution than what I had originally come up with.

**Comment by agitter** [comment]
Closed by #1104
